### PR TITLE
relational-operators: Recommend to use consistent relational operators

### LINKED ISSUE #31: Recommend to use similar relational operator in entire project.
# Relational Operators usage recommendation

I recommend using either `>=` / `<=` or `>` / `<` relationals operators on entire project.

### Scenario:
In front end, there are logics like `>=256 && <=512` and in other components/files or in backend  there is `>255 && <511`. 

### Benefits:
No high stress to the human mind. Searching for those ints in entire project will result accurate.

> My personal preference is to use `>=` or `<=` sign. e.g `foo.length >= 1`, `foo >= 256 && bar <= 512`.

### PR DESCRIPTION
**Pull request by sbimochan**
fixes #31 

Screenshot:

Screen Shot 2020-07-17 at 11 10 46

### CODE REVIEW

**Comment by anishmanandhar** [comment]
@sbimochan can you please attach screenshot also

File: docs/introduction.md [hunk 1]
```diff
@@ -131,3 +131,11 @@ title: General Coding Standards
 * Follow Gitflow.
 
 * Sanity checklist document should be present.
+
+* Consistent Relational Operators:
+
+  * Recommended to use either `>=` / `<=` or `>` / `<` relationals operators consistently in entire codebase.
```
Comment (kabirbaidhya) [inline docs/introduction.md:137]:
```suggestion
  * Use relational operators either `>=` / `<=` or `>` / `<`  consistently in entire codebase.
```

File: docs/introduction.md [hunk 2]
```diff
@@ -131,3 +131,11 @@ title: General Coding Standards
...
+
+  * Scenario: In front end, there are logics like `>=256 && <=512` and in other components/files or in backend  there is `>255 && <511`.
```
Comment (kabirbaidhya) [inline docs/introduction.md:139]:
```suggestion
  * Example: In some places, there are `>=256 && <=512` and in other place there is `>255 && <511`.
```

File: docs/introduction.md [hunk 3]
```diff
@@ -131,3 +131,11 @@ title: General Coding Standards
...
+
+  * Benefits: No high stress to human mind. Searching for those numbers in entire codebase will result accurate.
```
Comment (kabirbaidhya) [inline docs/introduction.md:141]:
```suggestion
  * Why? Lesser cognitive burden, enforce consistency and avoid confusion. 
```

**Comment by sbimochan** [comment]
It became better. Thanks @kabirbaidhya

**Pull request merged**